OpenTaiko/src/Stages/05.SongSelect/CActSortSongs.cs

```diff
@@ -9,16 +9,10 @@ public CActSortSongs() {
 			lci.Add(new CItemList(CLangManager.LangInstance.GetString("SONGSELECT_SORT_TITLE"), CItemBase.EPanelType.Normal, 0, "", "", new string[] { "Z,Y,X,...", "A,B,C,..." }));
 			lci.Add(new CItemList(CLangManager.LangInstance.GetString("SONGSELECT_SORT_SUBTITLE"), CItemBase.EPanelType.Normal, 0, "", "", new string[] { "Z,Y,X,...", "A,B,C,..." }));
 			lci.Add(new CItemList(CLangManager.LangInstance.GetString("SONGSELECT_SORT_LEVEL"), CItemBase.EPanelType.Normal, 0, "", "", new string[] { "13,12,11,...", "1,2,3,..." }));
-			//lci.Add( new CItemList( "Best Rank",	CItemBase.Eパネル種別.通常, 0, "", "", new string[] { "E,D,C,...",		"SS,S,A,..." } ) );
-			//lci.Add( new CItemList( "PlayCount",	CItemBase.Eパネル種別.通常, 0, "", "", new string[] { "10,9,8,...",		"1,2,3,..." } ) );
-
-			//lci.Add( new CItemList( "SkillPoint",	CItemBase.Eパネル種別.通常, 0, "", "", new string[] { "100,99,98,...",	"1,2,3,..." } ) );
```

Comment:
I think these sorting options might be potentially useful.

Reply:
As long as it is commented out, this code has no effect. We could bring the code to life with other mechanisms, but in my opinion currently it's nothing more than a small note that you can make as a developer, but has no place in the repo. I am of course open to other opinions and suggestions and am happy to bring the code back if necessary.

Reply:
I think that if it does work after uncommented, it should be instead made enabled on user option and not commented out. If it breaks or do weird things, of course remove it.